cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -212,6 +214,12 @@ func buildFlags(args *Args) []cli.Flag {
 			Usage:       "whether to validate integrity of image bootstrap",
 			Destination: &args.ValidateSignature,
 		},
+		&cli.StringFlag{
+			Name:        "api-socket",
+			Value:       defaultAPISocket,
+			Usage:       "path to the nydus-snapshotter HTTP server",
```

Comment:
`unix domain socket path to serve HTTP-based system management interface?`
In addition, can we also support `ip:port`? so that allows prometheus server to access metrics from remote.

Reply:
Sure. we can expose a IP based http server later

Reply:
fixed